sumologic/resource_sumologic_cse_first_seen_rule.go

```diff
@@ -0,0 +1,213 @@
+package sumologic
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"log"
+)
+
+func resourceSumologicCSEFirstSeenRule() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSumologicCSEFirstSeenRuleCreate,
+		Read:   resourceSumologicCSEFirstSeenRuleRead,
+		Delete: resourceSumologicCSEFirstSeenRuleDelete,
+		Update: resourceSumologicCSEFirstSeenRuleUpdate,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"baseline_type": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"baseline_window_size": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"category": {
```

Comment:
"category" is sort of deprecated at the API layer, and auto-calculated based on the tags provided, so we haven't exposed it in other TF rule resources. So I think we can remove it here as well.